rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -53,8 +53,7 @@ if(AMENT_ENABLE_TESTING)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
   endif()
 
-  add_executable(test_interfaces_cpp "test/test_interfaces.cpp")
-  add_dependencies(test_interfaces_cpp ${PROJECT_NAME} ${rosidl_generate_interfaces_TARGET})
```

Comment:
The removed target dependency broke the CI job: http://ci.ros2.org/job/ros2_batch_ci_osx/432/console#console-section-14

Fixed in fc65db0b5c1e0b2f017081fd70de32838b51521b